src/pages/myungjinlee/Main/Feed.js

```diff
@@ -0,0 +1,154 @@
+    this.setState({
+      commentList: CoomentInfo,
+      feedList: FeedData,
+    });
+  }
+  handleCommentValue = (e) => {
+    this.setState({
+      commentValue: e.target.value,
+    });
+  };
+
+  addComment = (e) => {
+    e.preventDefault();
+    const { commentList, commentValue } = this.state;
+    this.setState({
+      commentList: [
+        ...commentList,
+        {
+          id: commentList.length + 1,
+          userName: 'wecode',
+          content: commentValue,
+        },
+      ],
+      commentValue: '',
+    });
+  };
+
+  regeistComent = (e) => {
+    this.setState({
+      comment: e.target.value,
+    });
+  };
+  inputComment = () => {
+    const add = this.state.replies;
+    add.push(this.state.comment);
+    this.setState({
+      replies: this.state.replies,
+      comment: '',
+    });
+  };
+  PressClick = (e) => {
+    this.inputComment();
+  };
+  pressEnter = (e) => {
+    if (e.key === 'Enter') {
+      this.inputComment();
+    }
+  };
+  render() {
+    const { commentList, feedList } = this.state;
+    return (
+      <>
+        <div className="feed_top">
+          <ul className="feed">
+            <li className="profile_img">
+              <img className="cat" src={this.props.feedimg} alt="" />
+            </li>
+            <div className="name">
+              {this.props.userName}
+              <div className="nameSub">{this.props.subname}</div>
+            </div>
+            <li className="feed_right">
+              <i className="fas fa-ellipsis-h"></i>
+            </li>
+          </ul>
+          <div className="feed_img">
+            <img src={this.props.img} alt="" />
+          </div>
+        </div>
+        <div className="feed_comment">
+          <section className="feed_bottom">
+            <ul className="icons">
+              <li className="icons_left">
+                <i className="fas fa-heart"></i>
+                <i className="far fa-comment"></i>
+                <i className="far fa-paper-plane"></i>
+              </li>
+              <li className="icons_right">
+                <i className="far fa-bookmark"></i>
+              </li>
+            </ul>
+            <section className="comment">
+              <div className="profile_img_small">
+                <img className="hicat" src="" alt="" />
+              </div>
+              <span className="nickname">Wecode님 외 5명이 좋아합니다.</span>
+              <div className="text">
+                {this.props.content}
+                <button>...더 보기</button>
+                <a className="more" href="www.google.com">
+                  댓글 12개 모두 더보기
+                </a>
+              </div>
+            </section>
+          </section>
+          {commentList.map((comment) => {
+            return (
+              <Commentbox
+                key={comment.id}
+                name={comment.userName}
+                comment={comment.content}
+              />
+            );
+          })}
```

Comment:
```suggestion
          {commentList.map(comment => (
              <Commentbox
                key={comment.id}
                name={comment.userName}
                comment={comment.content}
              />
            )
          )}
```
arrow function에서
1. 인자가 하나일 경우에는 인자쪽은 () 생략할 수 있고
2. 함수 본문에서 return만 할 경우에는 {} 를 생략할 수 있습니다.

Reply:
아하 넵 수정하겠습니다. 알려주셔서 감사합니다 :)

Reply:
그런데 prettier 를 사용해서 인자가 하나일 경우에 저장하면 자동으로 ()가 생성됩니다. ㅠㅠ